Assignment3/process_json.py

```
@@ -0,0 +1,182 @@
+#!/usr/bin/python3
```

Ruff (0.14.2)

Shebang is present but file is not executable

(EXE001)